Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift

```diff
@@ -64,12 +79,14 @@ class DesktopPairingViewController: UITableViewController {
     }
 
     private func scan() {
-        let vc = QRCodeScannerViewController()
+        let vc = QRCodeScannerViewController(label: "Go to Gnosis Safe Web and select Connect wallet.")
```

Comment:
passing a String doesn't add the "bold" style to the "Gnosis Safe Web" and "Connect wallet" words. The solution would be to create and pass NSAttributedString (you can create its subclass, NSMutableAttributedString). Then, you assign that string to the label by `uiLabel.attributedText = attributedLabel`